File: Data Structures & Algorithms/search-in-rotated-sorted-array-ii/submission-5.ts
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {boolean}
     */
    search(nums: number[], target: number): boolean {
        for(let ele of nums){
            if(ele == target){
                return true;
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/sort-colors/submission-5.java
class Solution {
    public void sortColors(int[] nums) {
        Map<Integer,Integer> map = new HashMap<>();
        for(int ele : nums){
                map.put(ele,map.getOrDefault(ele,0) + 1);
        }
        int idx = 0;
        for(int i = 0;i<= 2;i++){
            int count = map.getOrDefault(i,0);
            for(int j = 0;j<count;j++){
                nums[idx++] = i;
            }
        }
    }
}